js/main.js

```diff
@@ -13,25 +12,82 @@ $imgURL.addEventListener('input', function (event) {
 $entryForm.addEventListener('submit', function (event) {
   event.preventDefault();
 
+  var entryFormInfo = {};
+
   entryFormInfo[$entryForm.elements.title.name] = $entryForm.elements.title.value;
   entryFormInfo[$entryForm.elements.imageUrl.name] = $entryForm.elements.imageUrl.value;
   entryFormInfo[$entryForm.elements.notes.name] = $entryForm.elements.notes.value;
```

Comment:
I would just do `entryFormInfo.propertyName`, no need to pull the name of the input, you already know the name since you are writing the code.